wolox-books/src/app/components/user/user.ts

```diff
@@ -1,8 +1,8 @@
 export interface User {
-    firstName: string;
-    lastName: string;
+    first_name: string;
+    last_name: string;
```

Comment:
Why this changes? I think this is not really needed and you should use `camelCase` for naming